bundle/manifests/crd-drop-status-validation-patch.yaml

```diff
@@ -0,0 +1,4 @@
+- op: replace
```

Comment:
we should exclude this from the bundle manifests file

Reply:
ignore this -- will fix as part of https://github.com/openshift/elasticsearch-operator/pull/446